zip-bolt-support.rst

```diff
@@ -0,0 +1,334 @@
+
+1.1 Customer and Merchant Signing Keys
+-------------
+
+The customer and the merchant both have key pairs from a suitable signature scheme. These are denoted as:
+``<cust-pk>, <cust-sk>`` and 
+``<merch-pk>, <merch-sk>``, respectively, where ``pk`` stands for "public key" and ``sk`` stands for the corresponding "secret key".
+
+The merchant must be able to issue blind signatures, so they have an additional keypair; this keypair is denoted as:
+``<MERCH-PK>, <MERCH-SK>``.
+
+The customer key pair is specific to the channel and must not be reused. The merchant key pair is long term and should be used for all customer channels. 
+
+1.2 Wallets
+-------------
+A Bolt channel allows a customer to make or receive a sequence of payments off chain. These payments are tracked and validated using a sequence of *wallets*. A wallet consists of the customer's public key (which ties the wallet to the channel), a wallet-specific public key (which can be from any suitable signature scheme), denoted ``<wpk>``, and the current customer and merchant balances.
+
+After each payment, the customer receives an updated wallet and blind signatures from the merchant on the wallet contents allowing channel close as specified below.
+
+1.2 Opening a Channel: Overview
+-------------
+To open a channel, the customer and merchant exchange key information and set the channel token ``<channel-token> = <cust-pk>, <merch-pk>, <MERCH-PK>``. 
+
+They agree on their respective initial balances ``initial-cust-balance`` and ``initial-merch-balance``.
+
+The customer picks an inital wallet public key ``<wpk>``.
+
+The customer and merchant escrow the necessary funds in a funding transaction, denoted ``escrow-tx``. 
+
+1.3 Closing a Channel: Overview
+-------------
+
+A customer should be able to close the channel by posting a *closing token* ``close-token``, which is a blind signature from the merchant under ``<MERCH-PK>`` on a special closing wallet that contains ``<cust-pk>, <wpk>, <balance-cust>, <balance-merch>, CLOSE``. We use ``cust-close-tx`` to denote the transaction posted by the customer to initiate channel closure.
+
+A merchant should be able to close the channel by either posting a special closing transaction ``merch-close-tx`` (detailed in Section 2.3.2) or, if the customer posts an outdated version of their closing token, a signed revocation token, ``revocation-token`` as detailed below. The revocation token ``revocation-token`` is a signature under the wallet public key ``<wpk>`` on the special revocation message ``<wpk> || REVOKED``. The transaction posted by the merchant to dispute is denoted ``dispute-tx``.
```

Comment:
Define what makes a closing token "outdated". (I think that tokens are associated with wallet versions, but it isn't clear.)